src/main/java/simulator/simbot/SimBot.java

```diff
@@ -27,5 +48,140 @@ public SimBotCommandCenter getCommandCenter() {
+                            serverSocket.getLocalPort() + "...");
+                    Socket server = serverSocket.accept();
+
+                    System.out.println("Just connected to " + server.getRemoteSocketAddress());
+
+                    String content;
+                    BufferedReader in = new BufferedReader(new InputStreamReader(server.getInputStream()));
+                    PrintWriter out = new PrintWriter(server.getOutputStream(), true);
+                    exit = false;
+
+                    while (run && !exit) {
+                        content = in.readLine();
+
+                        if (content != null) {
+                            double value = 0;
+                            if (!content.contains("WHEELS") && !content.contains("REGISTER") && !content.contains("GET")) {
+                                value = Double.parseDouble(content.substring(content.indexOf(':') + 1));
+                            }
+                            switch (content.substring(0, content.indexOf(':'))) {
+                                case "FORWARD":
+                                    //fl fr bl br
+                                    this.commandCenter.setWheelPower(value, value, value, value);
+                                    System.out.println("FORWARD " + value);
+                                    break;
+                                case "BACKWARD":
+                                    this.commandCenter.setWheelPower(-value, -value, -value, -value);
+                                    System.out.println("BACKWARD " + value);
+                                    break;
+                                case "RIGHT":
+                                    this.commandCenter.setWheelPower(value, -value, value, -value);
+                                    System.out.println("RIGHT " + value);
+                                    break;
+                                case "LEFT":
+                                    this.commandCenter.setWheelPower(-value, value, -value, value);
+                                    System.out.println("LEFT " + value);
+                                    break;
+                                case "WAIT":
+                                    System.out.println("WAITING FOR " + value + " SECONDS");
+                                    break;
+                                case "STOP":
+                                    this.commandCenter.setWheelPower(0, 0, 0, 0);
+                                    System.out.println("STOPPING");
+                                    break;
+                                case "KILL":
+                                    run = false;
+                                    System.out.println("Exiting\n");
+                                    break;
+                                case "GET":
+                                    String name = content.substring(content.indexOf(':') + 1);
```

Comment:
define  content.substring(content.indexOf(':') + 1); as String  value and refer to that string instead of parsing all over the place